From c86131164139fcb0ce1b448856ef452bdf6a346c Mon Sep 17 00:00:00 2001
From: Jason <r64343@freescale.com>
Date: Fri, 26 Mar 2010 16:45:37 +0800
Subject: [PATCH] ENGR00122046 Update DDR init script for MX53 EVK

Update DDR init script according to compass:
http://compass.freescale.net/go/216805297
FCP 7 KB 25-Mar-2010

setmem /32 0x53fa8570 = 0x00180000 ->
setmem /32 0x53fa8570 = 0x00200000
setmem /32 0x53fa8578 = 0x00180000 ->
setmem /32 0x53fa8578 = 0x00200000

Signed-off-by:Jason Liu <r64343@freescale.com>
---
 board/freescale/mx53_evk/flash_header.S |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/freescale/mx53_evk/flash_header.S b/board/freescale/mx53_evk/flash_header.S
index de5b7ec..459c01e 100644
--- a/board/freescale/mx53_evk/flash_header.S
+++ b/board/freescale/mx53_evk/flash_header.S
@@ -61,9 +61,9 @@ MXC_DCD_ITEM(2, IOMUXC_BASE_ADDR + 0x558, 0x00380040)
 MXC_DCD_ITEM(3, IOMUXC_BASE_ADDR + 0x560, 0x00380000)
 MXC_DCD_ITEM(4, IOMUXC_BASE_ADDR + 0x564, 0x00380040)
 MXC_DCD_ITEM(5, IOMUXC_BASE_ADDR + 0x568, 0x00380040)
-MXC_DCD_ITEM(6, IOMUXC_BASE_ADDR + 0x570, 0x00180000)
+MXC_DCD_ITEM(6, IOMUXC_BASE_ADDR + 0x570, 0x00200000)
 MXC_DCD_ITEM(7, IOMUXC_BASE_ADDR + 0x574, 0x00380000)
-MXC_DCD_ITEM(8, IOMUXC_BASE_ADDR + 0x578, 0x00180000)
+MXC_DCD_ITEM(8, IOMUXC_BASE_ADDR + 0x578, 0x00200000)
 MXC_DCD_ITEM(9, IOMUXC_BASE_ADDR + 0x57c, 0x00380040)
 MXC_DCD_ITEM(10, IOMUXC_BASE_ADDR + 0x580, 0x00380040)
 MXC_DCD_ITEM(11, IOMUXC_BASE_ADDR + 0x584, 0x00380000)
-- 
1.5.4.4

